### DISCUSSION TITLE
PVC in multi-cluster with VMware

### DISCUSSION THREAD

**Discussion by skyle52** [rancher/rke2#3700] [General]
Hello all,

Currently I have Rancher and rke running on on-premise VMware platform for auto provision and container storage. When PVC is created, it's create a vmdk in a storage pool in VMware, therefore, I cannot migrate the VM (rke node) to another cluster or running my rke in multi-cluster setup. Are there any solution that I can still keep my setup but able to migrate and run my in multi-cluster?

Thank you all!

**Comment by brandond** [comment]
I think you want to post this at https://github.com/rancher/rancher. you don't appear to be using RKE2?

**Reply by skyle52** [reply]
ok, thanks for the notice, i will move my question to appropriate group
